samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb

```diff
@@ -9,15 +9,15 @@
     "In this notebook you will create e2e mnist Kubeflow Pipeline to perfom:\n",
```

Comment:
<p>Hah, just noticed a typo that may as well be fixed. Can you update "perfom" on the first line to "perform"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/7355/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>